neqo-bin/src/udp.rs

```diff
@@ -58,22 +58,19 @@ impl Socket {
     }
 
     /// Send the UDP datagram on the specified socket.
-    pub fn send(&self, d: Datagram) -> io::Result<()> {
+    pub fn send(&self, d: &Datagram) -> io::Result<()> {
         let transmit = Transmit {
             destination: d.destination(),
             ecn: EcnCodepoint::from_bits(Into::<u8>::into(d.tos())),
-            contents: Vec::from(d).into(),
```

Comment:
I think we can then also remove `impl From<Datagram> for Vec<u8>`?

Reply:
:+1: https://github.com/mozilla/neqo/pull/1899/commits/0f68d88a52c50d213f61d8dbf574675fb2fc1f38.